projects/js-packages/charts/src/components/line-chart/stories/index.stories.tsx

```diff
@@ -15,6 +15,34 @@ const meta: Meta< StoryArgs > = {
 	title: 'JS Packages/Charts/Types/Line Chart',
 	argTypes: {
 		...lineChartMetaArgs.argTypes,
+		seriesCount: {
+			control: { type: 'radio' },
+			options: [ 'single', 'multiple', 'many' ],
+			description: 'Number of data series',
+			table: { category: 'Data' },
+		},
+		dimensionMode: {
```

Comment:
This one has no effect for me either and isn't a LineChart prop, maybe we need some typing on these args (to help AI?)